_site/js/sketch.js

```diff
@@ -482,15 +482,18 @@ function cubify(inputString) {
 						_this.camera
 					);
 
-					this.element.style.transform = `translate(${coords2d.x}px, ${coords2d.y}px) translate(-50%, 0)`;
+					this.element.style.setProperty("--posX", `${coords2d.x}px`);
+					this.element.style.setProperty("--posY", `${coords2d.y}px`);
 
 					dist = countDistanceToCamera(
 						this.parent.position,
 						_this.camera.position
 					);
 					tint = interpolate(dist, 1250, 1350, 255, 0);
-					this.element.style.color =
-						"rgb(" + tint + ", " + tint + ", " + tint + ")";
+
+					this.element.style.setProperty("--text-tint-r", tint);
+					this.element.style.setProperty("--text-tint-g", tint);
+					this.element.style.setProperty("--text-tint-b", tint);
```

Comment:
If `tint` has the same value for all, maybe just set it once, and apply that to all three channels?

So instead of 

```css
 color: rgb(var(--text-tint-r), var(--text-tint-g), var(--text-tint-b));
```

you would get


```css
 color: rgb(var(--tint), var(--tint), var(--tint));
```